doc/src/guides/solving/solving-guidance.md

```diff
@@ -0,0 +1,221 @@
+>>> solve(x**2 - pi, x, dict=True)
+[{x: -1.77245385090552}, {x: 1.77245385090552}]
+```
+
+To use exact values for numbers such as $6.2$ or $1/2$, refer to [Python numbers
+vs. SymPy Numbers](python-vs-sympy-numbers).
+
+In certain cases, using an inexact value will prevent SymPy from finding a
+result. For example, this exact equation can be solved:
+
+```py
+>>> from sympy import symbols, solve, sqrt
+>>> x = symbols('x')
+>>> eq = x**sqrt(2) - 2
+>>> solve(eq, x, dict=True)
+[{x: 2**(sqrt(2)/2)}]
+```
+
+but if you use the inexact equation `eq = x**1.4142135623730951 - 2`, SymPy will
+not return a result despite attempting for a long time. 
+
+## Options That Can Speed up {func}`~.solve`
+
+### Include Solutions Making Any Denominator Zero by Using `check=False`
+
+Normally, {func}`~.solve` checks whether any solutions make any denominator
+zero, and automatically excludes them. If you want to include those solutions,
+and speed up {func}`~.solve` (at the risk of obtaining invalid solutions), set
+`check=False`:
+
+```py
+>>> from sympy import Symbol, sin, solve
+>>> x = Symbol("x")
+>>> solve(sin(x)/x, x, dict=True) # 0 is excluded
+[{x: pi}]
+>>> solve(sin(x)/x, x, dict=True, check=False) # 0 is not excluded
+[{x: 0}, {x: pi}]
+```
+
+### Do Not Simplify Solutions by Using `simplify=False`
+
+Normally, {func}`~.solve` simplifies all but polynomials of order 3 or greater
+before returning them and (if `check` is not False) uses the general
+{func}`simplify <sympy.simplify.simplify.simplify>` function on the solutions
+and the expression obtained when they are substituted into the function which
+should be zero. If you do not want the solutions simplified, and want to speed
+up {func}`~.solve`, use `simplify=False`.
+
+```py
+>>> from sympy import solve
+>>> from sympy.abc import x, y
+>>> expr = x**2 - (y**5 - 3*y**3 + y**2 - 3)
+>>> solve(expr, x, dict=True)
+[{x: -sqrt(y**5 - 3*y**3 + y**2 - 3)}, {x: sqrt(y**5 - 3*y**3 + y**2 - 3)}]
+>>> solve(expr, x, dict=True, simplify=False)
+[{x: -sqrt((y + 1)*(y**2 - 3)*(y**2 - y + 1))}, {x: sqrt((y + 1)*(y**2 - 3)*(y**2 - y + 1))}]
+```
+
+## Parse a String Representing the Equation
+
+If you are creating the expression yourself, we [recommend against using parsing
+a string](
+https://github.com/sympy/sympy/wiki/Idioms-and-Antipatterns#strings-as-input).
+But if you are programmatically reading in a string, this approach is
+convenient.
+
+You can parse a string representing the equation into a form that SymPy can
+understand (for example, {any}`Eq <sympy.core.relational.Eq>` form), then apply
+solve the parsed expression. Parsing an equation from a string requires you to
+use {func}`transformations <sympy.parsing.sympy_parser.parse_expr>` for SymPy to
+- interpret equals signs
+- create symbols from your variables
+-  use more mathematical (rather than standard Python) notation, for example the
+exponent operator can be parsed from `^` rather than having to use Python's
+`**`.
```

Comment:
I guess as long as we're mentioning string parsing, we can mention quickly that SymPy supports [parsing LaTeX](https://docs.sympy.org/dev/modules/parsing.html#sympy.parsing.latex.parse_latex). In my experience, a lot of people find that feature useful. Perhaps it should also be mentioned in the "see also" for `parse_expr`.